templates/script/script.js

```diff
@@ -158,95 +138,89 @@ function createList(data)
         head_tr.appendChild(document.createElement("th")).innerText = null;
         head_tr.setAttribute('class', 'border-bottom');
         table.appendChild(head_tr);
-        document.getElementById("table_div").appendChild(table);
-        for(var i = 0; i < data_array.length; i++)
-        {
-            table.appendChild(document.createElement("tr")).id = "tr"+i;
-            document.getElementById("tr"+i).appendChild(document.createElement("td")).innerText = i+1;
-            document.getElementById("tr"+i).childNodes[0].setAttribute('class', 'bold');
-            document.getElementById("tr"+i).appendChild(document.createElement("td")).innerText = data_array[i].name;
-            document.getElementById("tr"+i).appendChild(document.createElement("td")).innerText = data_array[i].email;
-            document.getElementById("tr"+i).appendChild(document.createElement("td")).innerText = data_array[i].phone;
-            document.getElementById("tr"+i).appendChild(document.createElement("td")).id = "td"+i;
-            var rec_id = data_array[i].id;
-            var link = document.createElement('a');
+        divTable.appendChild(table);
+       
+        for (var i = 0; i < data_array.length; i++) {
```

Comment:
Please, read how 'var' and 'let' are different.
If you use 'let' when it's a nice idea to use it everywhere, instead of 'var'